app/scripts/modules/amazon/src/pipeline/stages/deployCloudFormation/evaluateCloudFormationChangeSetExecution.service.ts

```diff
@@ -0,0 +1,47 @@
+import { IPromise, module } from 'angular';
+import { EXECUTION_SERVICE, ExecutionService } from '@spinnaker/core';
+import { IExecution, IExecutionStage } from 'core/domain';
+import { Application } from 'core/application';
+
+export class EvaluateCloudFormationChangeSetExecutionService {
+  public static $inject = ['executionService'];
+  constructor(private executionService: ExecutionService) {}
+
+  private clearPipelineColor(): void {
```

Comment:
I didn't find any other way to change it, the conditional import of a less file in evaluateCloudFormationChangeSetExecutionMarkerIcon.tsx also comes from this 😞 

This stage is intended to mimic the ManualJudgement stage as under certain circumstances it will pause execution waiting for user feedback, IMHO from the UX perspective, orange color in Spinnaker is bond to that state that requires user action. 

Do you know any other way to achieve this?

Reply:
Not longer required with https://github.com/spinnaker/deck/pull/7671/commits/b088c914c75463bf966663c866624f61473cb9fa